openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1306,7 +1306,7 @@ def test_activation_email(self):
         sent_email = mail.outbox[0]
         self.assertEqual(sent_email.to, [self.EMAIL])
         self.assertEqual(sent_email.subject, "Activate Your edX Account")
-        self.assertIn("activate your account", sent_email.body)
+        self.assertIn("activating your edX account", sent_email.body)
```

Comment:
Would you mind interpolating `settings.PLATFORM_NAME` here instead of hardcoding `edX`? That ensures these tests continue to pass if the platform name is changed.